### ISSUE TITLE
leg shortening

### ISSUE THREAD

**Issue by uisch** [obophenotype/human-phenotype-ontology#3775]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
leg shortening

### 2. Synonyms
shortening of the leg; lower limb shortening

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
N/A

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP:0002814

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Includes all diseases characterized by mesomelic or rhizomelic leg shortening

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
N/A

**Comment by pnrobinson** [comment]
Please us this term
https://hpo.jax.org/app/browse/term/HP:0012106
or this term
https://hpo.jax.org/app/browse/term/HP:0004987

**Comment by uisch** [comment]
I of course noticed these terms, but in some cases, it is unclear whether it is 0012106 or 004987. So I think a more general term would be helpful.

**Comment by pnrobinson** [comment]
Maybe one of the terms near here?
https://hpo.jax.org/app/browse/term/HP:0009816
